python/cuml/svm/svc.pyx

```diff
@@ -177,6 +172,10 @@ class SVC(SVMBase, ClassifierMixin):
         n_samples / (n_classes * n_samples_of_class[i])
     max_iter : int (default = 100*n_samples)
         Limit the number of outer iterations in the solver
+    multiclass_strategy : str ('ovo' or 'ovr', default 'ovo')
+        Multiclass classification strategy. 'ovo' uses
+        `OneVsOneClassifier <https://scikit-learn.org/stable/modules/generated/sklearn.multiclass.OneVsOneClassifier.html>`_ # noqa: E501
+        while 'ovr' selects `OneVsRestClassifier <https://scikit-learn.org/stable/modules/generated/sklearn.multiclass.OneVsRestClassifier.html>`_ # noqa: E501
```

Comment:
These comments are showing up in the docs too:
![image](https://user-images.githubusercontent.com/42954918/100830500-0d802880-3421-11eb-850b-e164d6557201.png)